frontend/src/assets/Text.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Text = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
Are these svgs for Text and View component really needed? I would expect that there should be some out of the box components that you could use ...

Reply:
Ooooh, I see now they are actually some images, maybe the use Icon suffix, like TextIcon and ViewIcon

Reply:
I'll use the icon suffix, it's a better naming convention for our icons.